x/genutil/types/genesis.go

```diff
@@ -68,7 +67,7 @@ func (ag *AppGenesis) ValidateAndComplete() error {
 	}
 
 	if ag.GenesisTime.IsZero() {
-		ag.GenesisTime = cmttime.Now()
+		ag.GenesisTime = time.Now().Round(0).UTC()
```

Comment:
TM has this as an helper, should we just create it in the SDK instead and use it at the remaining places?

Reply:
na, we want to avoid more sdk deps 😄

Reply:
Then we should add a rule in https://github.com/crypto-com/cosmos-sdk-codeql 😅 
This exists: https://github.com/crypto-com/cosmos-sdk-codeql/blob/main/src/system-time.ql, which is good, but it should not trigger with `time.Now().Round(0).UTC()` and advise to use `time.Now().Round(0).UTC()` otherwise.

Reply:
Created https://github.com/crypto-com/cosmos-sdk-codeql/pull/8.
cc @yihuang.